docs/how-to/custom-env.txt

```diff
@@ -171,17 +171,19 @@ image (using the ``FROM`` instruction).
 Here is an example of a ``Dockerfile`` that installs both ``conda``- and
 ``pip``-based dependencies.
 
-.. code::
+.. code:: bash
 
    FROM determinedai/environments:cuda-10.2-pytorch-1.7-tf-1.15-gpu-0.10.0
-   RUN apt-get update && apt-get install -y unzip python-opencv graphviz
-   COPY environment.yml /tmp/environment.yml
-   COPY pip_requirements.txt /tmp/pip_requirements.txt
-   RUN conda env update --name base --file /tmp/environment.yml && \
-       conda clean --all --force-pkgs-dirs --yes
+   RUN apt-get update
+   RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
+   RUN apt-get install -y unzip python-opencv graphviz
+   COPY environment.yml /tmp/environment.yml # Optional
+   COPY pip_requirements.txt /tmp/pip_requirements.txt # Optional
+   RUN conda env update --name base --file /tmp/environment.yaml # Optional
+   RUN conda clean --all --force-pkgs-dirs --yes
    RUN eval "$(conda shell.bash hook)" && \
-       conda activate base && \
-       pip install --requirement /tmp/pip_requirements.txt
+       conda activate base
+   RUN pip install --requirement /tmp/pip_requirements.txt # Optional
```

Comment:
I think breaking this into a separate RUN would actually break things: https://pythonspeed.com/articles/activate-conda-dockerfile/

Reply:
updated